fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -84,8 +84,13 @@
     // FIXITY
 
     public static final Resource FIXITY_TYPE = createResource(PREMIS_NAMESPACE + "Fixity");
+
+    public static final Property HAS_MESSAGE_DIGEST_ALGORITHM =
+            createProperty(PREMIS_NAMESPACE + "hasMessageDigestAlgorithm");
```

Comment:
Here is one: http://id.loc.gov/ontologies/premis.html#p_hasMessageDigestAlgorithm
There is also a `messageDigestAlgorithm` on page 25 of the mentioned PDF.
Maybe this is another Premis version?